src/utils/noop.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * No operation function
+ */
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export const noop = (): void => {};
```

Comment:
we prefer function declaration than const-arrow
```suggestion
export function noop(): void {};
```